lintro/ai/review/lint_bridge.py

```
@@ -0,0 +1,128 @@
+            tool_option_dict={},
+            exclude=None,
+            include_venv=False,
+            incremental=False,
+            action=Action.CHECK,
+            post_tools=set(),
+            auto_install=False,
+            lintro_config=lintro_config,
+        )
+        try:
+            result = tool.check(paths=changed_files, options={})
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)